Some tests fail if pytest is run in root folder

Example of failure:
```
_______________________________________ test_exhaustive_template_is_isomorphic ________________________________________

    def test_exhaustive_template_is_isomorphic():
>       g1 = Graph().parse("043_exhaustive_example_perfect_output.ttl")

tests\test_template043.py:43:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
.venv\lib\site-packages\rdflib\graph.py:1234: in parse
    source = create_input_source(
.venv\lib\site-packages\rdflib\parser.py:326: in create_input_source
    ) = _create_input_source_from_location(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

file = None, format = None, input_source = None, location = '043_exhaustive_example_perfect_output.ttl'

    def _create_input_source_from_location(file, format, input_source, location):
        # Fix for Windows problem https://github.com/RDFLib/rdflib/issues/145 and
        # https://github.com/RDFLib/rdflib/issues/1430
        # NOTE: using pathlib.Path.exists on a URL fails on windows as it is not a
        # valid path. However os.path.exists() returns false for a URL on windows
        # which is why it is being used instead.
        if os.path.exists(location):
            location = pathlib.Path(location).absolute().as_uri()

        base = pathlib.Path.cwd().as_uri()

        absolute_location = URIRef(location, base=base)

        if absolute_location.startswith("file:///"):
            filename = url2pathname(absolute_location.replace("file:///", "/"))
>           file = open(filename, "rb")
E           FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\dlinke\\MyProg_exp040\\043_exhaustive_example_perfect_output.ttl'

.venv\lib\site-packages\rdflib\parser.py:373: FileNotFoundError
```
I'll send a PR with a fix.

--- a/tests/test_template040.py
+++ b/tests/test_template040.py
@@ -58,7 +58,9 @@ def test_empty_template():
 
 # isomorphic tests
 def test_exhaustive_template_is_isomorphic():
-    g1 = Graph().parse("040_exhaustive_example_perfect_output.ttl")
+    g1 = Graph().parse(
+        Path(__file__).parent / "040_exhaustive_example_perfect_output.ttl"
+    )
     g2 = convert.excel_to_rdf(
         Path(__file__).parent / "040_exhaustive_example.xlsx", output_type="graph"
     )
@@ -66,7 +68,9 @@ def test_exhaustive_template_is_isomorphic():
 
 
 def test_minimal_template_is_isomorphic():
-    g1 = Graph().parse("040_minimal_example_perfect_output.ttl")
+    g1 = Graph().parse(
+        Path(__file__).parent / "040_minimal_example_perfect_output.ttl"
+    )
     g2 = convert.excel_to_rdf(
         Path(__file__).parent / "040_minimal_example.xlsx", output_type="graph"
     )

--- a/tests/test_template041.py
+++ b/tests/test_template041.py
@@ -40,7 +40,9 @@ def test_simple():
 
 
 def test_exhaustive_template_is_isomorphic():
-    g1 = Graph().parse("041_exhaustive_example_perfect_output.ttl")
+    g1 = Graph().parse(
+        Path(__file__).parent / "040_exhaustive_example_perfect_output.ttl"
+    )
     g2 = convert.excel_to_rdf(
         Path(__file__).parent / "041_exhaustive_example.xlsx", output_type="graph"
     )

--- a/tests/test_template043.py
+++ b/tests/test_template043.py
@@ -40,7 +40,9 @@ def test_simple():
 
 
 def test_exhaustive_template_is_isomorphic():
-    g1 = Graph().parse("043_exhaustive_example_perfect_output.ttl")
+    g1 = Graph().parse(
+        Path(__file__).parent / "040_exhaustive_example_perfect_output.ttl"
+    )
     g2 = convert.excel_to_rdf(
         Path(__file__).parent / "043_exhaustive_example.xlsx", output_type="graph"
     )

